.changeset/olive-berries-work.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/blocks-admin": minor
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add the method `extractTextContents` to extract text contents of blocks and `replaceTextContents` to import and replace text contents of blocks.
```

Comment:
Changelog should not only contain changed APIs, it should also describe user visible changes/features